src/coffee/services/s3client.coffee

```diff
@@ -220,10 +220,14 @@ class S3Client
           Compress.compressImage(tmp_resized, tmpDir, extension)
       .then =>
         @sendMetrics 'increment', 'image.resized'
-        header = 'x-amz-acl': 'public-read'
+        if isNaN expire
+          expire = 2592000
```

Comment:
For merkur we need to set cache-control, max-age header for images for optimization.
After setting the expiration headers a browser will only make requests to the server if it does not have the resource in its cache.